src/Internal/Metadata/Cip25/V2.purs

```diff
@@ -142,6 +143,7 @@ instance DecodeAeson Cip25V2 where
       2 -> pure Cip25V2
       _ -> Left $ TypeMismatch "Cip25V2"
 
+-- Why not an instance of ToMetadata?
```

Comment:
It's not useful alone, so it was done that way to prevent confusion between `Cip25Metadata` and `Cip25MetadataEntry` (the users do not need to deal with data-representations of a single entry, because the standard only specifies the encoding for Cip25Metadata, that is very different from our representation in PS (k/v object vs. an array)). Please remove this comment, or write a new one based on this clarification.